core/arch/x86/proc.c

```diff
@@ -162,6 +166,31 @@ get_cache_sizes_intel(uint max_val)
     }
 }
 
+static void
+get_xstate_area_offsets(bool has_kmask, bool has_zmm_hi256, bool has_hi16_zmm)
+{
+    int cpuid_res_local[4]; /* eax, ebx, ecx, and edx registers (in that order) */
+    if (has_kmask || has_zmm_hi256 || has_hi16_zmm) {
```

Comment:
Put inside DOLOG to ensure no execution w/o LOG.

Reply:
Done.